repository/src/org/pentaho/platform/security/userroledao/jackrabbit/AbstractJcrBackedUserRoleDao.java

```diff
@@ -886,4 +893,17 @@ private boolean canDeleteRole( Session session, final IPentahoRole role ) {
     }
     return usersToBeRemoved.toArray( new String[0] );
   }
+
+  public boolean getUseJackrabbitUserCache() {
```

Comment:
The correct naming convention is to use "isUseJackrabbitUserCache". The "is*" prefix is for boolean accessors.